identity-enabler/deviceId-mobile-app/src/components/CredentialHeader.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+    import { IOTA_IDENTITY_RESOLVER } from "../config";
+    import { shortenDID } from "../lib/ui/helpers";
+
+    export let credential;
+    export let color = "black";
+    export let hideDetails = false;
+</script>
+
+<div id="wrapper" style="color: {color};">
+    <i class="icon-credential" />
+    <h2>{credential.type[1]}</h2>
+    {#if !hideDetails}
+        <p>
+            <span>Issued to </span>
+            <a href="{IOTA_IDENTITY_RESOLVER}/{credential.credentialSubject.id}" class="emphasis" target="_blank">
+                {shortenDID(credential.credentialSubject.id)}
+            </a>
+            <span> by </span>
+            <a href="{IOTA_IDENTITY_RESOLVER}/{credential.issuer.id}" class="emphasis" target="_blank"
+                >{credential.issuer.name}
+            </a>
+            <span> at </span>
+            <span class="emphasis">{new Date(credential.issuanceDate).toLocaleString()}</span>
```

Comment:
probably you need to be more explicit, specifying a locale, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString